### DISCUSSION TITLE
Sample Apps - Where did they go?

### DISCUSSION THREAD

**Discussion by NickBCastano** [appium/appium#18212] [General]
Hi everyone, Im running thru a course on Appium and the class asks me to download the sample apps in the `/sample-apps/apps/` folder but that folder seems to be gone now. 

Are these sample apps gone for good?

**Comment by KazuCocoa** [comment]
Perhaps https://github.com/appium/appium/tree/1.x/sample-code , which is for old Appium 1.x
